iris_grib/tests/__init__.py

```diff
@@ -35,6 +36,33 @@
 #: Basepath for iris-grib loadable test files.
 _TESTDATA_PATH = os.path.join(os.path.dirname(__file__), 'testdata')
 
+override = os.environ.get("GRIB_TEST_DATA_PATH")
+if override:
+    if os.path.isdir(os.path.expanduser(override)):
+        _TESTDATA_PATH = os.path.abspath(override)
+
+
+def skip_grib_data(fn):
+    """
+    Decorator to choose whether to run tests, based on the availability of
+    external data.
+
+    Example usage:
+        @skip_data
+        class MyDataTests(tests.IrisGribTest):
+            ...
+
+    """
+    dpath = _TESTDATA_PATH
+    evar = "GRIB_TEST_NO_DATA"
+    no_data = not os.path.isdir(dpath) or os.environ.get(evar)
+
+    skip = unittest.skipIf(
+        condition=no_data, reason="Test(s) require external data."
```

Comment:
"Test(s) require external data." is a bit brief, and indistinguishable from the _Iris_ missing-testdata message.
? Could we go with ...
```
    reason = ("Test(s) require test grib files."
              "\nSee : https://github.com/SciTools/iris-grib/tree/v0.15.1/iris_grib/tests/testdata")
```

Reply:
@pp-mo No worries. Good idea. I've changed it to something different, but similar:
```
Test(s) require missing external GRIB test data.
```
I think it's a bit noisy to have a URI in there... anyways, see what you think :+1: